src/main/java/it/gov/pagopa/pu/fileshare/controller/IngestionFlowFilesController.java

```diff
@@ -68,4 +69,10 @@ public ResponseEntity<Resource> downloadIngestionFlowErrorsFile(Long organizatio
       .body(fileResource);
   }
 
+  @Override
+  public ResponseEntity<SignedUrlResultDTO> downloadNotice(Long organizationId, Long ingestionFlowFileId) {
```

Comment:
This API is supposed to return the file, not the signedUrl...
is the openApi be wrong?
Once obtained the signedUrl, you should call it (I think it's a GET) and return it's result to the calling client

ask to @RiccardoGiuliani  details about the signedUrlResultDTO